BlitzBoard/Features/HomeScreenViewController.swift

```diff
@@ -1,16 +1,57 @@
+// LeagueTableViewController.swift
+// BlitzBoard
 //
-//  HomeScreenViewController.swift
-//  BlitzBoard
-//
-//  Created by Kiasha Rangasamy on 2024/05/06.
+// Created by Kiasha Rangasamy on 2024/04/11.
 //
 
 import UIKit
 
-class HomeScreenViewController: UIViewController {
-
+class LeagueTableViewController: UIViewController, ViewModelDelegate {
+    
+    func reloadView() {
+    }
+    
+    func show(error: String) {
+    }
+    
+    // MARK: IBOutlets
+    @IBOutlet private weak var tableView: UITableView!
+    @IBOutlet private weak var wallpapers: UIImageView!
```

Comment:
can put a new line after the last outlet

follow this format

//All IB outlets

//All IB actions

//All public vars

//All private vars

e.g.
// MARK: IBOutlets
  (new line here)
    @IBOutlet private weak var tableView: UITableView!
    @IBOutlet private weak var wallpapers: UIImageView!
      (new line here)
    // MARK: IBActions
    ....
      (new line here)
     // MARK: Variables
       (new line here)
    var exampleOne: Int
    var exampleTwo: int
    (new line here)
    private var exampleThree: Int

Reply:
keep this format for all future PR's